extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -164,6 +167,14 @@ export abstract class DacFxConfigPage extends BasePage {
 		// return rootpath of opened folder in file explorer if one is open, otherwise default to user home directory
 		return vscode.workspace.rootPath ? vscode.workspace.rootPath : os.homedir();
 	}
+
+	protected validateFilePath() {
```

Comment:
>validateFilePath [](start = 11, length = 16)

I think this would be better as something like appendFileExtensionIfNeeded() - validate usually doesn't involve actually modifying anything so some would consider it unexpected behavior that this does